tests/acceptance/test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+import _ from 'lodash';
+import {expect} from 'chai';
+import webdriverio from 'webdriverio';
+import path from 'path';
+
+const BROWSER_OPTIONS = {
+  desiredCapabilities: {
+    browserName: 'PhantomJS'
+  }
+};
+const VIEWPORT_SIZE = {
+  width: 1024,
+  height: 768
+};
+const URL = 'file://' + path.join(__dirname, '..', '..', 'visual-report',
+            'statics', 'index.html');
+
+describe('Mocha Mugshot acceptace tests', function() {
+  let client;
+
+  before(function(done) {
+    client = webdriverio.remote(BROWSER_OPTIONS).init()
+      .setViewportSize(VIEWPORT_SIZE)
+      .url(URL)
+      .then(function() {
+        done();
+      });
+  });
+
+  it('should display the navigation bar', function(done) {
+    client
+        .isExisting('.navbar')
+        .then(function(exists) {
+          expect(exists).to.be.true;
+          done();
+        });
+  });
+
+  it('should display the results', function(done) {
+    client
+        .isExisting('.results')
+        .then(function(exists) {
+          expect(exists).to.be.true;
+          done();
+        });
+  });
+
+  it('should toggle the details when clicking on a test title', function(done) {
+    client
+        // clicks on the first test title
```

Comment:
Is this because WDIO will only click on the first element matched by a selector? If so, using the `:first` pseudo-selector will make things may more clear.